### PR TITLE
[Merged by Bors] - chore: golf using omega

### PR DESCRIPTION
**Pull request by grunweg**
Backported from #11314.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by jcommelin — APPROVED** [review]
Thanks :tada: 

If CI passes, please remove the label `awaiting-CI` and merge this yourself, by adding a comment `bors r+`.

bors d+

**Comment by grunweg** [comment]
Thank you for the fast review. mathlib builds, so let's
bors r+

**Comment by grunweg** [comment]
Let me try again.
bors r+
